splunk_add_on_ucc_framework/app_manifest.py

```diff
@@ -88,6 +88,9 @@ def manifest(self) -> Dict[str, Any]:
     def update_addon_version(self, version: str) -> None:
         self._manifest["info"]["id"]["version"] = version
 
+    def add_ucc_version(self, version: str) -> None:
```

Comment:
We don't need this one anymore after the Slack thread?

Reply:
Agreed, adding version to GlobalConfig is enough.